doc/classes/AnimationTree.xml

```diff
@@ -185,12 +45,154 @@
 			The number of possible simultaneous sounds for each of the assigned AudioStreamPlayers.
 			For example, if this value is [code]32[/code] and the animation has two audio tracks, the two [AudioStreamPlayer]s assigned can play simultaneously up to [code]32[/code] voices each.
 		</member>
+		<member name="last_process_pass" type="int" setter="set_last_process_pass" getter="get_last_process_pass" default="1">
+		</member>
 		<member name="process_callback" type="int" setter="set_process_callback" getter="get_process_callback" enum="AnimationTree.AnimationProcessCallback" default="1">
 			The process mode of this [AnimationTree]. See [enum AnimationProcessCallback] for available modes.
 		</member>
+		<member name="property_map" type="Dictionary" setter="set_property_map" getter="get_property_map" default="{}">
+		</member>
+		<member name="root_motion_position" type="Vector3" setter="set_root_motion_position" getter="get_root_motion_position" default="Vector3(0, 0, 0)">
+			Retrieve the motion delta of position with the [member root_motion_track] as a [Vector3] that can be used elsewhere.
```

Comment:
Some of the text here is still for a function just getting the value, needs a new wording

Reply:
My English is not goodt. Although I update the description, it need review.